jekyll/_cci2/ecs-ecr.md

```diff
@@ -346,51 +296,99 @@ jobs:
+          container-image-name-updates: 'container=${MY_APP_PREFIX}-service,tag=stable'
+workflows:
   deploy:
-    # ...
+    jobs:
+      - update-tag
+```
+
+#### Verify Successful Deployment of Service Update to ECS
+
+Once you have deployed the service update to the ECS service, you should verify that the update was successful. The orb shown below enables you to cerify that your update was made to the ECS revision.
+
+```yaml
+version: 2.1
+orbs:
+  aws-cli: circleci/aws-cli@0.1.4
+  aws-ecs: circleci/aws-ecs@0.0.3
+jobs:
+  verify-deployment:
+    docker:
+      - image: 'circleci/python:3.7.1'
     steps:
-      # ...
-      - run:
-          name: Push image to ECR
-          command: |
-            . venv/bin/activate
-            eval $(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)
-            docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPOSITORY_NAME:$CIRCLE_SHA1
+      - aws-cli/install
+      - aws-cli/configure:
+          aws-access-key-id: $AWS_ACCESS_KEY_ID
+          aws-region: $AWS_REGION
       - run:
-          name: Deploy
-          command: |
-            . venv/bin/activate
-            export ECS_TASK_FAMILY_NAME="${AWS_RESOURCE_NAME_PREFIX}-service"
-            export ECS_CONTAINER_DEFINITION_NAME="${AWS_RESOURCE_NAME_PREFIX}-service"
-            export EXECUTION_ROLE_ARN="arn:aws:iam::$AWS_ACCOUNT_ID:role/${AWS_RESOURCE_NAME_PREFIX}-ecs-execution-role"
-            ./deploy.sh
-      - run:
-          name: Test deployment (Please manually tear down AWS resources after use, if desired)
-          command: |
-            . venv/bin/activate
-            TARGET_GROUP_ARN=$(aws ecs describe-services --cluster $ECS_CLUSTER_NAME --services $ECS_SERVICE_NAME | jq -r '.services[0].loadBalancers[0].targetGroupArn')
-            ELB_ARN=$(aws elbv2 describe-target-groups --target-group-arns $TARGET_GROUP_ARN | jq -r '.TargetGroups[0].LoadBalancerArns[0]')
-            ELB_DNS_NAME=$(aws elbv2 describe-load-balancers --load-balancer-arns $ELB_ARN | jq -r '.LoadBalancers[0].DNSName')
-            curl http://$ELB_DNS_NAME | grep "Hello World!"
+          name: Get last task definition
+          command: >
+            TASK_DEFINITION_ARN=$(aws ecs describe-task-definition \
+                --task-definition ${MY_APP_PREFIX}-service \
+                --output text \
+                --query 'taskDefinition.taskDefinitionArn')
+            echo "export TASK_DEFINITION_ARN='${TASK_DEFINITION_ARN}'" >>
+            $BASH_ENV
+      - aws-ecs/verify-revision-is-deployed:
+          family: '${MY_APP_PREFIX}-service'
+          cluster-name: '${MY_APP_PREFIX}-cluster'
+          task-definition-arn: '${TASK_DEFINITION_ARN}'
+workflows:
+  test-workflow:
+    jobs:
+      - verify-deployment
 ```
 
 ### Set Up a Workflow
 
-Use workflows
-to link the `build` and `deploy` jobs.
+Use workflows to link the `build` and `deploy` jobs.
```

Comment:
The Full Configuration File section below and possibly the workflows section should probably be updated to reflect the above changes at least?

Reply:
Yeah, I am going to work through the document with Tom T. on Thursday and we are going to update/remove relevant sections. I expect to have this completed by Friday.

Reply:
Cutting down on the length of the tutorial would be great! Coincidentally I had just received feedback from a user: "there is only one line in the requirements.txt, can't we just add it to this config? Going through the docs is a pain for learning how to deploy on ECS already."

I'm wondering if it would be good to make this simple orb demo branch the master branch instead? https://github.com/CircleCI-Public/circleci-demo-aws-ecs-ecr/tree/simple_orb_demo